hw/ip/flash_ctrl/doc/checklist.md

```diff
@@ -76,7 +76,7 @@ Code Quality  | [CDC_SYNCMACRO][]       | Done        |
 
  Type         | Item                    | Resolution  | Note/Collaterals
 --------------|-------------------------|-------------|------------------
-Security      | [SEC_CM_IMPLEMENTED][]  | Not Started |
+Security      | [SEC_CM_IMPLEMENTED][]  | Done        |
```

Comment:
If this is also done, then should it be D2S?

Reply:
I suppose the [RTL review](https://docs.google.com/presentation/d/1F8gk-WcxwF-g2mn29A_ko0De8amiyv965Od3FKJCj0U/edit?disco=AAAAR_D9z_w) for D2S is still pending as per our OT milestones slides.

Reply:
yeah that's right. It's "done" from an implementation standpoint, but not done from the milestone perspective.